applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -136,10 +138,10 @@ def _check_results(self,mp,A,b):
             d = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT)
             for i in range(3):
                 if abs(u[i]) > 0.0:
-                    error = (d[i] - u[i])/u[i]
-                    if error > 1.0e-6:
+                    error = abs((d[i] - u[i])/u[i])
```

Comment:
BTW, this is a bug in the check

Reply:
because of not using the `std::abs` ?

Reply:
Because not using abs (could give false negative if the error is negative)